# Ricky Doucette

## Research question/interests

Ricky: I am interested in this dataset because I am a fan of hockey and like to see the statistical aspects of players' performances. I would like to find if there is a correlation between drops and increases in certain statistics for consecutive years for the top NHL players in points. Specific examples of research questions I would like to discover answers to are: Does an increase/decrease in time on ice per game relate to an increase in certain stats like overall points in the season? Do the top players that have a decrease in their offensive stats in a successive season make up for it with an increase in their defensive stats? How many of the top 250 players in points from the 2017-2018 season managed to make the top 350 players in points for the 2018-2019 season? How many of those players didn't make the top 350 just because they retired?

To get more specific, the research questions I would primarily like to answer are:
1. Does the average change in a top NHL player's offensive stats from one season to the next have a negative correlation with their defensive stats? That is if a player has a decrease in say points, do their stats like +/- and blocked shots tend to increase?
2. Does an increase in time on ice per game significantly increase a player's overall statistics? We could define significance as an extra point per additional minute of time on ice per game. So say a player plays 82 games both seasons. If in the first season they score 100 points with 18 minutes of TOI and the following season they score 115 points with 18.5 minutes of TOI, we could deem their increase in ice time as a significant factor in their increase in points. The calculations to determine this would be as follows:

In [1]:
import pandas as pd
nhl2017 = pd.read_csv('../data/processed/Data301ProjectData2017.csv')
nhl2018 = pd.read_csv('../data/processed/Data301ProjectData2018.csv')

In [1]:
s1 = (82*18)/100
s1

14.76

In [3]:
s2 = (82*18.5)/115
s2

13.191304347826087

In [4]:
s1-s2

1.568695652173913

We see here that in this example, the player was getting on average 1 point per 14.76 minutes they played in the first season, then 1 point per 13.19 minutes played in the second season. This is an increase in 1.57 points per minutes played.
If this player were to have had their TOI increased to 18.5 from 18, at the same pace of 1 point per 14.76 minutes played, they would have had about 103 points in the following season